library/Notifications/Widget/ItemList/EventListItem.php

```diff
@@ -105,14 +107,15 @@ protected function assembleTitle(BaseHtmlElement $title): void
 
         /** @var Objects $obj */
         $obj = $this->item->object;
-        $name = $obj->getName();
+        $name = (new DeferredText(function () use ($obj) {
```

Comment:
`DefferedText` is not required here, as the name has already been rendered. Please undo the change.